aws/data_source_aws_pricing_product_test.go

```diff
@@ -0,0 +1,120 @@
+package aws
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccDataSourceAwsPricingProduct_ec2(t *testing.T) {
+	os.Setenv("AWS_DEFAULT_REGION", "us-east-1")
```

Comment:
We should save the original value and reset it after completion or this could break other tests:

```go
	oldRegion := os.Getenv("AWS_DEFAULT_REGION")
	os.Setenv("AWS_DEFAULT_REGION", "us-east-1")
	defer os.Setenv("AWS_DEFAULT_REGION", oldRegion)
```

The same should be applied to the other acceptance test. Admittedly, we need to remove using environment variables for this purpose. 🙁